### PR TITLE
fix: SPM resources and example project setup

### PR DESCRIPTION
**Pull request by harry-brown-cko**
## Proposed changes

- Resources now loading correctly through SPM
- Example SPM project now uses local version of Frames rather than version on Github
- Use version number from constants rather than from `Bundle`
    - This is because the version number is not available on the bundle with SPM

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [X] Bugfix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [X] Lint and unit tests pass locally with my changes
* [X] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

## Further comments

None

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,8 +1,9 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.3
```
Comment (harry-brown-cko) [inline Package.swift:1]: had to bump the version here to enable explicit resource declarations

File: Package.swift [hunk 2]
```diff
@@ -21,13 +22,59 @@ let package = Package(
...
+                .process("Resources/arrows/keyboard-down.png"),
+                .process("Resources/schemes/icon-dinersclub@2x.png")
+            ]
         ),
-        .testTarget(
-            name: "Frames-Tests",
-            dependencies: ["Frames"],
-            path: "Tests"
-        )
+// Tests are currently broken as Mockingjay doesn't support SPM
```
Comment (harry-brown-cko) [inline Package.swift:63]: currently blocked from running tests against our SPM package as Mockingjay does not support SPM

File: Source/Extensions/StringExtensions.swift [hunk 3]
```diff
@@ -4,8 +4,12 @@ import UIKit
 extension String {
 
     private func getBundle(forClass: AnyClass) -> Bundle {
+        #if SWIFT_PACKAGE
+        let baseBundle = Bundle.module
+        #else
```
Comment (harry-brown-cko) [inline Source/Extensions/StringExtensions.swift:9]: recommended bundle access in SPM

File: Source/UI/Views/SchemeIconsStackView.swift [hunk 4]
```diff
@@ -17,7 +17,7 @@ class SchemeIconsStackView: UIStackView {
     func addSchemeIcon(scheme: CardScheme) {
         let imageView = UIImageView()
         imageView.setContentHuggingPriority(.defaultHigh, for: .horizontal)
-        let image = "schemes/icon-\(scheme.rawValue)".image(forClass: SchemeIconsStackView.self)
+        let image = "icon-\(scheme.rawValue)".image(forClass: SchemeIconsStackView.self)
```
Comment (harry-brown-cko) [inline Source/UI/Views/SchemeIconsStackView.swift:20]: this is the fix for the images

**Review by deepesh-vasthimal-cko — APPROVED** [review]

**Pull request merged**